### ISSUE TITLE
Stop loss not working 

### ISSUE THREAD

**Issue by baselito** [CryptoGnome/LickHunterPRO#4]
Not sure if it's fixed but in alpha 0.52 stop loss didn't work. 

https://cdn.discordapp.com/attachments/704578798200422411/750293243429912626/unknown.png

**Comment by CryptoGnome** [comment]
Are you able to send me the log of this along with anymore details

**Comment by baselito** [comment]
auto_qty = True

percentBal = 0.8
maxPosition = 90

longQty = 0.1
shortQty = 0.1

long_vwap_offset = 3.7
short_vwap_offset = 3.7

leverage = 8

lick_value = 5

take_profit = 1.13
stop_loss = 3

[error.log](https://github.com/CryptoGnome/LickHunterPRO/files/5159425/error.log)

the stop loss didn't execute so it's still in play

forgot to mention it's eth on bybit

**Comment by CryptoGnome** [comment]
Thats perfect i will look at this ASAP and get back to you
